kcidb/test_orm.py

```diff
@@ -1,5 +1,7 @@
 """kcdib.orm module tests"""
 
+# Over 1000 lines, pylint: disable=too-many-lines
```

Comment:
The lines exceed 1000. I hope this is okay?

Reply:
Yeah, for tests we can kinda tolerate it, but thanks for caring about it :+1: